ha/k8s_setup/ha_setup.py

```diff
@@ -180,11 +180,17 @@ def process(self):
             # TBD delete once data_pod_label is avilable from confstore
             data_pod_label = 'cortx-data'
 
+            # Dummy value fetched for now. This will be replaced by the key/path for the service ID/machine ID onces that is avilable in confstore
+            # Ref ticket EOS-???
+            machine_id_key = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}product_release')
+            # TBD delete once resource_id_key is avilable from confstore
```

Comment:
resource_id_key replace machine_id_key

Reply:
dd comment as this 'status/podIP' is placeholder key the key for machine id will be replace it once available